docs/resources/vpc_peering_connection_accepter_v2.md

```diff
@@ -101,3 +101,11 @@ All of the argument attributes except accept are also exported as result attribu
 * `peer_vpc_id` - The VPC ID of the accepter tenant.
 
 * `peer_tenant_id` - The Tenant Id of the accepter tenant.
+
+## Import
+
+VPC Peering Connection Accepter can be imported using the `id`, e.g.
```

Comment:
it's better to import the resource with **vpc_peering_connection_id** because  the `id` of `flexibleengine_vpc_peering_connection_accepter` is the connection id.